docs/pod-metrics.md

```diff
@@ -9,6 +9,8 @@
 | kube_pod_labels | Gauge | Kubernetes labels converted to Prometheus labels | | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `label_POD_LABEL`=&lt;POD_LABEL&gt; <br> `uid`=&lt;pod-uid&gt; | STABLE |
 | kube_pod_status_phase | Gauge | The pods current phase | | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `phase`=&lt;Pending\|Running\|Succeeded\|Failed\|Unknown&gt; <br> `uid`=&lt;pod-uid&gt; | STABLE |
 | kube_pod_status_ready | Gauge | Describes whether the pod is ready to serve requests | | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `condition`=&lt;true\|false\|unknown&gt; <br> `uid`=&lt;pod-uid&gt; | STABLE |
+| kube_pod_status_ready_time | Gauge | Time when pod passed readiness probes. | seconds | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `uid`=&lt;pod-uid&gt; | STABLE |
+| kube_pod_status_containers_ready_time | Gauge | Time when pod containers entered Ready state. | seconds | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `uid`=&lt;pod-uid&gt; | STABLE |
```

Comment:
```suggestion
| kube_pod_status_container_ready_time | Gauge | Time when pod the container entered Ready state. | seconds | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `uid`=&lt;pod-uid&gt; | STABLE |
```

Would suggest to use singular here, since we already have `kube_pod_container_info` and `kube_pod_container_status_waiting`

Reply:
Hello @mrueg,
The k8s documentation specifies this particular condition as 
>ContainersReady: all containers in the Pod are ready. 

, hence the plural. Due to the fact there can be more than 1 containers in a pod and this condition turns into true only when **all** pods are ready, I think we should stick with plural. **What do you think?** If you still want to stick with singular that's fine for me as well, but I found it a little bit confusing.

Reply:
Thanks, good point! I'll defer to @lilic @brancz and @tariq1890 if we want to stick closer to k8s or not

Reply:
I would prefer keeping to how we have it in kube-state-metrics which is `kube_pod_container_info`, but up to @brancz and @mrueg .